src/sentry/integrations/bitbucket/issues.py

```diff
@@ -70,6 +71,11 @@ def get_create_issue_config(self, group, **kwargs):
     def get_link_issue_config(self, group, **kwargs):
         repo_choices, default_repo, issues = self.get_repo_choices(**kwargs)
```

Comment:
I'm changing this in a separate PR, we don't need `get_repo_choices` also returning issues (which we're now not even using anymore)